tests/modules/services/window-managers/sway/sway-default.conf

```diff
@@ -71,6 +71,7 @@ bindsym Mod1+w layout tabbed
 
 
 
+
```

Comment:
I wonder if it's worth changing stripping extra newlines out of the config, as at the moment the generated config doesn't look nice. @rycee does any other module do this?

Reply:
@alexarice Typically, this is done by concatenating a list of strings. I think it should be pretty easy to change in this case, though. The module uses a lot of code along the lines of
```nix
''
  ${concatStringsSep "\n" (mapAttrsToList inputStr input)}
  ${concatStringsSep "\n" (mapAttrsToList outputStr output)}
  ${concatStringsSep "\n" (mapAttrsToList modeStr modes)}
''
```
which produces these empty lines, if the code instead looks something like
```nix
''
  ${concatStringsSep "\n" (
      mapAttrsToList inputStr input
      ++ mapAttrsToList outputStr output
      ++ mapAttrsToList modeStr modes
  )}
''
```
then it shouldn't produce empty lines.

Reply:
Would it make sense to have this be part of a different PR?